cmd/vclusterctl/cmd/delete.go

```diff
@@ -144,7 +156,11 @@ func (cmd *DeleteCmd) Run(cobraCmd *cobra.Command, args []string) error {
 		err = client.CoreV1().Namespaces().Delete(context.Background(), cmd.Namespace, metav1.DeleteOptions{})
 		if err != nil {
 			if !kerrors.IsNotFound(err) {
-				return errors.Wrap(err, "delete namespace")
+				if cmd.IgnoreNotFound{
```

Comment:
Same as above - I don't think this is necessary as the line above ignores the "NotFound" error.